torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10226,6 +10241,47 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                # GradcheckError: gradcheck expects all tensor inputs are dense when check_sparse_nnz is set to False
                DecorateInfo(unittest.skip("Skipped!"), 'TestFwdGradients', 'test_forward_mode_AD'),
            )),
+    OpInfo('sparse.mm',
+           dtypes=floating_types_and(torch.bfloat16),
+           variant_test_name='reduce',
+           supports_autograd=True,
+           supports_out=False,
+           supports_gradgrad=False,
+           supports_forward_ad=False,
+           sample_inputs_func=sample_inputs_sparse_mm_reduce,
+           decorators=[onlyCPU],
+           skips=(
```

Comment:
@pearu - Do we still need to skip all of these?

Reply:
Yes, they still fails.

Reply:
@pearu could you please help review this one ? thanks!